README.md

```diff
@@ -152,6 +152,33 @@ console.log(JSON.stringify(results, null, 4));
 // ]
 ```
 
+### Positional information
```

Comment:
@philsturgeon @tbarn 
Added some piece of code to explain how positional information can be obtained.
Let me know if it's all clear for you.

There is no reasonable way to produce ranges (lines and columns) for already parsed content / object literals, e.g.
```js
const myObj = {
  foo: 'bar',
};
```

We cannot really produce anything meaningful in such case, therefore no range will be attached unless all the steps described below are taken.

Reply:
See https://github.com/stoplightio/spectral/pull/106#discussion_r268898760, I think we should consider just re-parsing if an object is passed in.